subsys/net/lib/wifi_mgmt_ext/wifi_mgmt_ext.c

```diff
@@ -74,7 +72,7 @@ static int __stored_creds_to_params(struct wifi_credentials_personal *creds,
 	params->security = creds->header.type;
 	params->mfp = WIFI_MFP_OPTIONAL;
 	params->channel = WIFI_CHANNEL_ANY;
-	params->timeout = CONNECTION_TIMEOUT;
+	params->timeout = CONFIG_WIFI_MGMT_EXT_CONNECTION_TIMEOUT;
```

Comment:
Commit log mentions increasing the timeout to 20s while the default value of `CONFIG_WIFI_MGMT_EXT_CONNECTION_TIMEOUT` is 30s. Please check once.

Reply:
I have cherry picked this from main, its a typo, should be `by 20s`, we can leave it as is.